### PR TITLE
Fixes #1866: Adds color formatting to /mail sendall

### LINKED ISSUE #1866: Colors wont work on /mail sendall tag. (sendall)
**EssentialsX version:** 2.0.1-b601

**Server software:** Spigot 1.12.2

**Server log:** Not... There is no error. 

**EssentialsX config:** Not needed.

### Details

Colors wont work and i just showed as "&4&lBELANGRIJK:" instead the bold & red BELANGRIJK. I do have OP. And the colors work when you send it to an player. (It only not work with SendAll)

If you are not enable to reproduce this issue. Please say what you need to retry it.

### PR DESCRIPTION
**Pull request by caojohnny**
**Issue synopsis - #1866**
The `/mail sendall` command did not allow users to use ampersand (`&`) formatted color codes in their messages. This is contrary to the `/mail send` command which does allow users to input said color codes.

**Proposed changes**
The same convention for sending a colored message as from `/mail send` is used in `/mail sendall`.

**Demo**

### CODE REVIEW

**Review by mdcfe — COMMENTED** [review]
This looks good, but I haven't built and tested it so I'm not going to explicitly approve this yet.

**Comment by caojohnny** [comment]
I appreciate the response nonetheless, thanks!

**Comment by Remski01** [comment]
Bump, 

(PS: Sorry if i did annoy you with this "bump")
(It just looks you kinda forget this :()

**Comment by caojohnny** [comment]
Bumping doesn't do anything. The maintainers have been busy, they look through all the issues and PRs whenever they're online.

**Comment by Remski01** [comment]
That's why i said Sorry between the ( ) under it. If i annoy them with bumping i atleast said sorry ;)

**Comment by mdcfe** [comment]
@DutchUser Bumping and continuing to reply doesn't help, because all its doing is adding to the many notifications we all get and obscuring potentially high priority tickets. We can merge this once someone has tested and approved it.

**Review by LaxWasHere — APPROVED** [review]

**Review by mdcfe — APPROVED** [review]

**Comment by caojohnny** [comment]
Thanks!

**Comment by mdcfe** [comment]
@AgentTroll No, thank *you* for the PR :p

**Pull request merged**
